src/settings_internal.js

```diff
@@ -137,9 +137,6 @@ var WASM_BINARY_FILE = '';
 // name of the file containing the Wasm Worker *.ww.js, if relevant
 var WASM_WORKER_FILE = '';
 
-// name of the file containing the Audio Worklet *.aw.js, if relevant
-var AUDIO_WORKLET_FILE = '';
```

Comment:
Is this a separate change?

Reply:
No, its just an internal setting that is no longer needed after this change.